tailwind.config.ts

```diff
@@ -203,7 +203,7 @@ const config = {
           0px -3px 2px 0px rgba(0, 0, 0, 0.14) inset`,
         // TODO: From current theme. Deprecate for 'button-hover'
         primary: "4px 4px 0px 0px hsla(var(--primary))",
-        "button-hover": "4px 4px 0 0 hsla(var(--primary-low-contrast))",
+        "button-hover": "4px 4px 0 0 hsla(var(--orange-900))",
```

Comment:
We're approaching a point where we have no more references to the orange color palette, and it is not present in the DS... The full yellow color palette is being imported in a separate branch. After that, we can migrate this to `--yellow-900` and should be able to remove the entire orange palette.